### ISSUE TITLE
[Lang] Translate all `range` into `ti.ndrange`

### ISSUE THREAD

**Issue by archibate** [taichi-dev/taichi#869]
**Concisely describe the proposed feature**
I found the following two code pieces same effect:
```py
for i in range(n):
  print(i)
```
```py
for i in ti.ndrange(n):
  print(i)
```
Seems `ti.ndrange` also supports 1D range?
Then why not we rename all `ti.ndrange` by `range`? Like:
```py
for i, j in range(n, m):
  print(i * 10 + j)
```
Yeah such syntax is **not correct in python**, but I believe taichi **doesn't have to be** syntax correct to match python, or I'll speak **struct-for** as an example: `for i, j in pixel`.
This will definitely be a big step to reduce learning barrier.

**Describe the solution you'd like (if any)**
Translate all `range` into `ti.ndrange` in `transform.py`.

**Additional comments**

**Comment by archibate** [comment]
I realized `range(2, 3)` represents `ti.ndrange((2, 3))` after all. Let's move to `range((2, 3))` and further discussion is required.

**Comment by yuanming-hu** [comment]
> I realized `range(2, 3)` represents `ti.ndrange((2, 3))` after all. Let's move to `range((2, 3))` and further discussion is required.

Right, `range((2, 3))` seems clearer to me then `range(2, 3)`, which already has a meaning in native Python.

**Comment by archibate** [comment]
I found this a little hard due to the current signle-pass transformer..
